File: tests/eval_c/214.c
#include <assert.h>
#include <string.h>

void f214(const char* s) {
    char a = s[0];
    char b = s[1];
    char t[3] = {a, b, '\0'};
    assert(strcmp(t, "ab") == 0);
}

File: benchmarks/oopsla25/subjects/214.py
import sys

def f214(s: str):
  a, b = s
  t = a + b
  assert t == "ab"

if __name__ == '__main__':
  my_string = ""
  if len(sys.argv) == 1:
    my_string = sys.stdin.read()
  else:
    fd = open(sys.argv[1], 'r')
    my_string = fd.read()
    fd.close()
  
  f214(my_string)
